File: assignment 10 - C# practice problems/3.cs
//Write a c# program to check two given numbersInput 
//and return true if one of the number is 50 or if their sum is 50.  

using System;
using System.Collections.Generic;
using System.Globalization;

namespace MyNameSpace
{
    class Program3
    {
        public static void CheckIfSumOrNumberIs50DriverFunction()
        {

            Console.WriteLine("Enter two numbers :");
            double num1 = double.Parse(Console.ReadLine());
            double num2 = double.Parse(Console.ReadLine());

            if (num1 == 50 || num2 == 50 || (num1 + num2).ToString() == $"{50:N0}")
            {
                Console.WriteLine("true");
            }
        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/3.js
//Write a JavaScript program to check two given numbersInput 
//and return true if one of the number is 50 or if their sum is 50.  
function checkIfSumOrNumberIs50DriverFunction() {
    let numbersInput = prompt("Enter two numbers seperated by a space");

    if (numbersInput != null) {
        numbersInput = numbersInput.trim();
        numbersInput = numbersInput.split(" ");

        let num1 = parseFloat(numbersInput[0]);
        let num2 = parseFloat(numbersInput[1]);

        if (num1 === 50 || (num1 + num2) === +(50).toFixed(0)) {
            alert("true");
        }
    }
}

checkIfSumOrNumberIs50DriverFunction();